02week/pigLatin.js

```diff
@@ -8,9 +8,56 @@ const rl = readline.createInterface({
 });
 
 
-function pigLatin(word) {
+function pigLatin(input) {
+  let inputStr = input.split('');
+  let counter  = 0;
 
-  // Your code here
+  inputStr.forEach(isVowel);
+  function isVowel(item,index){
+    if(index == 0)
+      console.log('Input letters are:');
+    console.log(item);
+  }
+  let index;
+  let newStr1;
```

Comment:
you're using a lot of lets and opening the scope, what's a better way to do this?

Reply:
Pushed new changes with refactored code

Reply:
https://github.com/bireswarb/javascript-workbook/pull/7